clang/test/dpct/device001.cu

```diff
@@ -160,6 +160,23 @@ int main(int argc, char **argv) {
   if(deviceProp.deviceOverlap){
   //dosomething.
   }
+
+  // CHECK: int Max1d = deviceProp.get_image1d_max();
+  int Max1d = deviceProp.maxTexture1D;
```

Comment:
please add lit test for this case maxTexture1D be reassign

Reply:
fixed